files/lib/system/discord/DiscordApi.class.php

```diff
@@ -1218,9 +1218,13 @@ public function createGuild($params)
      *
      * @return  array
      */
-    public function getGuild()
+    public function getGuild(bool $counts = false)
     {
-        $url = sprintf('%s/guilds/%s', $this->apiUrl, $this->guildID);
```

Comment:
Why you removed this?
Bette version would be this:
```PHP
$url = sprintf('%s/guilds/%s', $this->apiUrl, $this->guildID);
if ($withCounts) {
    $url .= '?with_counts=1';
}
```

Reply:
You are right, I may should not make PRs when I am tired 🫡 I will change it and update the PR when back at home.